src/pages/spawn/SpawnHost.tsx

```diff
@@ -23,14 +23,17 @@ export const SpawnHost = () => {
     MyHostsQuery,
     MyHostsQueryVariables
   >(GET_MY_HOSTS, {
-    pollInterval,
+    pollInterval: DEFAULT_POLL_INTERVAL,
     onError: (e) => {
       dispatchToast.error(
         `There was an error loading your spawn hosts: ${e.message}`
       );
     },
   });
-  usePolling(startPolling, stopPolling, refetch);
+  const migrationInProcess = !!data?.myHosts.find(
+    ({ volumes }) => !!volumes.find(({ migrating }) => migrating)
+  );
+  usePolling(startPolling, stopPolling, refetch, migrationInProcess);
```

Comment:
An options object would make this way more clear that `migrationInProcess` is the trigger for faster polling.